ui/angular.json

```diff
@@ -9,10 +9,15 @@
       "projectType": "application",
       "prefix": "app",
       "i18n": {
-        "sourceLocale": "en-US",
+        "sourceLocale": {
+          "code": "en",
```

Comment:
Nice to see that you can apply "en" for source locale - I just tried that, but without a JSON object and it failed. Good to see that it works, now. 👍